cli/cfncluster/cfncluster.py

```diff
@@ -24,6 +24,15 @@
 
 from . import cfnconfig
 
+#
+# configuration
+#
+username_map = { 'alinux' : 'ec2-user',
```

Comment:
This information is also available in cfn template. Can we have util function to figure mapping from template ?

Reply:
Yes. See above comment